application/src/main/scala/com/gu/deskcomexporttool/HttpClient.scala

```diff
@@ -29,7 +32,8 @@ object HttpClient {
 
       for {
         sttpResponse <- EitherT.right(sttpRequest.send())
-        body <- EitherT.fromEither(sttpResponse.body).leftMap(HttpError)
+        _ = log.debug(s"Received desk.com response status:${sttpResponse.code} body:${sttpResponse.body.right.getOrElse("").take(200)}")
```

Comment:
Magic 200?

Reply:
these are just truncating bodies for log messages, ill make it more explicit